### ISSUE TITLE
Run Visual Metrics on Windows

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/browsertime#1203]
There's a couple of things that needs to be fixed to be able to use visual metrics on Windows:
* [ ] Fix FFMPEG to work on Windows - will be fixed #1202 
* [ ] Chrome gets a popup on Windows: https://stackoverflow.com/questions/23087724/chromedriver-disable-developer-mode-extensions-pop-up-on-selenium-webdriver-au/44636414
* [ ] Installing imagemagick through choco doesn't include mogrify, installing it and choosing legacy commands fixes that. Either we add that to the docs or look for better solution.
* [ ] Verify and test that Visual Metrics works.

**Comment by brennie** [comment]
Not sure if this is in scope for this, but using Browsertime on an Android device from Windows presently does not work due to a bug in geckodriver/mozdevice (https://bugzilla.mozilla.org/show_bug.cgi?id=1610629) which I am presently working on.

**Comment by soulgalore** [comment]
Ah cool @brennie that sounds good. At the moment I'm mostly interested in getting everything to work in Travis, so I know we don't break things on updates. But getting Android to work is nice.

**Comment by dpalmeiro** [comment]
I am not sure how you plan on recording the screen, but for what it's worth the last time we tried to do this with GDI it caused a lot of overhead.  Perhaps, there is a more efficient solution through directshow or some other method.

**Comment by soulgalore** [comment]
Yeah but I think it will add some value even though it adds overhead, as long as it has more or less stable overhead. Also it fit into my long-term goal to get the Microsoft team more involved in Browsertime :D

**Comment by soulgalore** [comment]
Visual Metrics works in the new [GitHub Action for Windows](https://github.com/sitespeedio/browsertime/actions?query=workflow%3A%22Windows+Edge%22).
